### PR TITLE
add iOS build benchmarks

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Add a benchmark to measure the build time of various iOS assets when updating configurations. ensure that assemble (from the work in progress PR #32816) changes doesn't regress iOS build times

See also: https://github.com/flutter/flutter/pull/35481

### CODE REVIEW

File: dev/devicelab/bin/tasks/build_benchmark.dart [hunk 1]
```diff
@@ -8,5 +8,5 @@ import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/tasks/build_benchmarks.dart';
 
 Future<void> main() async {
-  await task(createBuildbenchmarkTask());
+  await task(createAndroidBuildbenchmarkTask());
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/build_benchmark.dart]: nit: capitalize Benchmark?

File: dev/devicelab/bin/tasks/build_benchmark_ios.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/tasks/build_benchmarks.dart';
+
+Future<void> main() async {
+  await task(createIosBuildbenchmarkTask());
```
Comment (goderbauer) [inline dev/devicelab/bin/tasks/build_benchmark_ios.dart]: same nit.

File: dev/devicelab/lib/tasks/build_benchmarks.dart [hunk 3]
```diff
@@ -10,63 +10,73 @@ import 'package:path/path.dart' as path;
...
-      final Map<String, double> allResults = <String, double>{};
-      allResults['first_build_debug_millis'] = initialBuildMilliseconds.toDouble();
-      allResults['second_build_debug_millis'] = secondBuildMilliseconds.toDouble();
-      allResults['after_config_change_build_debug_millis'] = thirdBuildMilliseconds.toDouble();
-      return TaskResult.success(allResults, benchmarkScoreKeys: allResults.keys.toList());
-    });
+    return createBuildCommand('apk');
   };
 }
+/// Creates a device lab build benchmark for iOS.
```
Comment (goderbauer) [inline dev/devicelab/lib/tasks/build_benchmarks.dart]: nit: add a blank line above this?

**Review by goderbauer — APPROVED** [review]
LGTM

File: dev/devicelab/bin/tasks/build_benchmark.dart (hunk 1 above)
Comment (jonahwilliams) [reply dev/devicelab/bin/tasks/build_benchmark.dart]: Done

File: dev/devicelab/bin/tasks/build_benchmark_ios.dart (hunk 2 above)
Comment (jonahwilliams) [reply dev/devicelab/bin/tasks/build_benchmark_ios.dart]: Done

File: dev/devicelab/lib/tasks/build_benchmarks.dart (hunk 3 above)
Comment (jonahwilliams) [reply dev/devicelab/lib/tasks/build_benchmarks.dart]: Done

**Pull request merged**